src/App.js

```diff
@@ -1,25 +1,17 @@
-import logo from './logo.svg';
+/* eslint-disable */
```

Comment:
- [ ] Please kindly do not disable linters, fix linters manually helps programmers to write clean and clear codebase 😁 😉